rdwatch/core/models/site_evaluation.py

```diff
@@ -44,6 +44,13 @@ class SiteEvaluation(models.Model):
         help_text="Polygon from this site's Site Feature",
         srid=3857,
         spatial_index=True,
+        null=True,
+    )
+    point = PointField(
+        help_text="Point from this site's Site Feature",
+        srid=3857,
+        spatial_index=True,
+        null=True,
```

Comment:
I think we should add a DB constraint to enforce that one of `geom` or `point` is non-null